saas/fsm/user_status_change/README.md

```diff
@@ -0,0 +1,29 @@
+# SaaS Manager - User Status Change
+
+## What it does
+
+This policy will raise an alert when Flexera SaaS Manager identifies users whose status in the HR roster changes to inactive.
```

Comment:
product doesn't want to use alert. instead change **raise an alert** to **create an incident.**